src/model/primitives/xhtml.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "string",
+    "description": "Limited xhtml content (Escaped XHTML)"
```

Comment:
I can't understand this primitive usage. Do we have some examples of what is allowed and what isn't?

Reply:
@fox1t No I don't have examples, this is up for discussion if you want to have it now. Perhaps this isn't required for the new version of HR. I just lifted it from the legacy RAML files and thought I should include it here.

Reply:
OK. I think we can remove it for the moment and add it in future if needed. @jackcmeyer what do you think?

Reply:
sounds good. I will remove it and refresh the PR unless @jackcmeyer has other thoughts?

Reply:
@fox1t @MichaelDalyDev would this be used to store rich text content? Such as content from a rich text editor where we would want to preserve the html that is generated?

Reply:
@jackcmeyer that sounds like a valid use-case, but I cannot confirm how this type was used in the legacy system. Perhaps we can remove it for now, and introduce something more appropriate when we need it?

What do you think?

Reply:
I agree. We can add it when we need it!

Reply:
@jackcmeyer ok cool. I will remove it and refresh the PR.